vcr/cassettes/spec/services/claims/fee_calculator/unit_price_spec.yml

```diff
@@ -1604,4 +1604,44 @@ http_interactions:
       string: '{"count":1,"next":null,"previous":null,"results":[{"id":58840,"scenario":5,"advocate_type":null,"fee_type":54,"offence_class":"K","scheme":2,"unit":"DAY","fee_per_unit":"0.00000","fixed_fee":"349.47000","limit_from":0,"limit_to":null,"modifiers":[],"strict_range":false}]}'
     http_version: 
   recorded_at: Sat, 03 Nov 2018 21:49:42 GMT
+- request:
```

Comment:
not a blocker - odd that this caused a new request recording. Any idea why? The unit_price_spec should be setup to always make the same requests (and therefore existing recordings should cover it)

Reply:
This must have a been a lurking cassette from a different branch... also :(